lib/wibox/widget/slider.lua

```diff
@@ -89,7 +89,10 @@ local slider = {mt={}}
 -- @propemits true false
 -- @propbeautiful
 
---- The cursor icon while grabbing the handle. The available cursor names are located in "/usr/include/X11/cursorfont.h" without the XC_ prefixes.
+--[[
```

Comment:
Please verify your changes locally before pushing, because now this property will not show up at all.

The first line in doc comments must start with `---` (three dashes). And block comments are not supported in doc comments, it must all be line comments.

Reply:
Thanks I'll fix it. But how can I verify the documentation code output locally?

Reply:
I also added two more comments under `@tparam[opt] string args.handle_cursor` so they would describe it. Would they show up? I didn't find anything related in the LDoc documents.
```lua
-- @tparam[opt] number args.handle_border_width The handle border width.
-- @tparam[opt] string args.handle_cursor
-- The cursor icon while grabbing the handle.
-- The available cursor names are located in "/usr/include/X11/cursorfont.h" without the XC_ prefixes.
-- @tparam[opt] gears.shape args.bar_shape The bar (background) shape.
-- @tparam[opt] number args.bar_height The bar (background) height.```

Reply:
the best way to find it out to build the docs locally as sclu suggested above

Reply:
I built the docs locally. They docs are rendered correctly, but unfortunately, ldoc doesn't seem to write the two lines of comment in front of `handle_cursor` and `beautiful.slider_handle_cursor` descriptions under "Object properties" and "Theme variables"

Reply:
Maybe the line length limit should be changed, so we can write longer descriptions in one line.

Reply:
> ldoc doesn't seem to write the two lines of comment in front of `handle_cursor` and `beautiful.slider_handle_cursor` descriptions under "Object properties" and "Theme variables"

That's by design. Those tables are a table of contents, and specifically only show the first sentence (and shouldn't be gamed by making one extra long sentence). The full text is displayed when you scroll/jump to actual content section.